app/templates/index.html

```diff
@@ -46,6 +48,205 @@ <h4>LET’S CONNECT</h4>
+                <div class="section-four-bg"></div>
+                <div class="section-block">
+                    <div class="section-title"><h2>Multi-Post Stories</h2></div>
+                    <div class="section-frame">
+                        <div class="cannopy">CANOPY
+                            <div class="section-img">
+                                <img src="/app/static/images/Counter.svg" alt="">
+                            </div>
+                        </div>
+                        <div class="backend">Back End Dev
+                            <div class="section-img">
+                                <img src="/app/static/images/Counter.svg" alt="">
+                            </div> 
+                        </div>
+                        <div class="year">2015</div>
+                    </div>
+                    <div class="section-primary-text">
+                        <p>
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <div class="section-tags">
+                        <button type="submit">html</button>
+                        <button type="submit">css</button>
+                        <button type="submit">javascript</button>
+                    </div>
+                    <div class="section-action">
+                        <button type="submit">See project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+    
+        <!--About me section-->
+        <section class="section-elements">
+            <div class="section-about">
+                <div class="section-block about">
+                    <div class="about-me-text">
+                        <h1>About <br> Myself</h1>
+                    </div>
+                    <div class="about-text">
+                        <p>I’m a software developer! I can help you build a product , 
+                            feature or website Look through some of my work and experience! 
+                            If you like what you see and have a project you need coded, 
+                            don’t hestiate to contact me.
+                        </p>
+                    </div>
+                    <div class="social-media">
+                        <h4>LET’S CONNECT</h4>
+                        <ul class="icons">
+                            <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                            <li><a href="#"></a><i class="fa-brands fa-linkedin-in"></i></li>
+                            <li><a href="#"><i class="fa-solid fa-m"></i></a></li>
+                            <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+                            <li><a href="#"><img src="/app/static/images/Vector.svg" alt=""></a></li>
+                        </ul>
```

Comment:
Great job using an ordered list for this section :clap: . 
- [ ] Kindly let's follow the order of the icons as they are shown in the Figma design. Also, for some reason, one of the icons is not showing on my end. 

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/185557596-d4debca7-f550-4260-b5b3-19cd3815fe9e.png)

**This is how it is supposed to be:**

![image](https://user-images.githubusercontent.com/86318284/185557689-d1573057-fee9-483c-80d5-7f57093d5ce1.png)

Reply:
Thanks @Nemwel-Boniface. 
Let me work on the changes.